nntrainer/tensor/float_tensor.h

```diff
@@ -186,180 +192,203 @@ class FloatTensor : public TensorBase {
+   * @copydoc Tensor::erf(Tensor &output)
    */
-  TensorV2 &erf(TensorV2 &output) const override;
+  Tensor &erf(Tensor &output) const override;
 
   /**
-   * @copydoc TensorV2::sin(TensorV2 &out, float alpha)
+   * @copydoc Tensor::sin(Tensor &out, float alpha)
    */
-  void sin(TensorV2 &out, float alpha) override;
+  void sin(Tensor &out, float alpha) override;
 
   /**
-   * @copydoc TensorV2::cos(TensorV2 &out, float alpha)
+   * @copydoc Tensor::cos(Tensor &out, float alpha)
    */
-  void cos(TensorV2 &out, float alpha) override;
+  void cos(Tensor &out, float alpha) override;
 
   /**
-   *  @copydoc TensorV2::dot(TensorV2 const &input, TensorV2 &output, bool
+   * @copydoc TensorBase::inv_sqrt(Tensor &out)
+   */
+  void inv_sqrt(Tensor &out) override;
+
+  /**
+   *  @copydoc Tensor::dot(Tensor const &input, Tensor &output, bool
    * trans, bool trans_in, float beta)
    */
-  TensorV2 &dot(TensorV2 const &input, TensorV2 &output, bool trans,
-                bool trans_in, float beta) const override;
+  Tensor &dot(Tensor const &input, Tensor &output, bool trans, bool trans_in,
+              float beta) const override;
 
   /**
-   * @copydoc TensorV2::dropout_mask(float dropout)
+   * @copydoc Tensor::dropout_mask(float dropout)
    */
   void dropout_mask(float dropout) override;
 
   /**
-   * @copydoc TensorV2::filter_mask(const TensorV2 &mask_len, bool reverse)
+   * @copydoc Tensor::filter_mask(const Tensor &mask_len, bool reverse)
    */
-  void filter_mask(const TensorV2 &mask_len, bool reverse) override;
+  void filter_mask(const Tensor &mask_len, bool reverse) override;
 
   /**
-   * @copydoc TensorV2::zoneout_mask(TensorV2 &opposite, float zoneout)
+   * @copydoc Tensor::zoneout_mask(Tensor &opposite, float zoneout)
    */
-  void zoneout_mask(TensorV2 &opposite, float zoneout) override;
+  void zoneout_mask(Tensor &opposite, float zoneout) override;
 
   /**
-   * @copydoc TensorV2::split(std::vector<size_t> sizes, int axis)
+   * @copydoc Tensor::split(std::vector<size_t> sizes, int axis)
    */
-  std::vector<TensorV2> split(std::vector<size_t> sizes, int axis) override;
+  std::vector<Tensor> split(std::vector<size_t> sizes, int axis) override;
 
   /**
-   * @copydoc TensorV2::cat(const std::vector<TensorV2> &tensors, int axis)
+   * @copydoc Tensor::cat(const std::vector<Tensor> &tensors, int axis)
    */
-  static TensorV2 cat(const std::vector<TensorV2> &tensors, int axis);
+  Tensor concat(const std::vector<Tensor> &tensors, int axis) override;
 
   /**
-   * @copydoc TensorV2::copy(const TensorV2 &from)
+   * @copydoc Tensor::copy(const Tensor &from)
    */
-  void copy(const TensorV2 &from);
+  void copy(const Tensor &from);
 
   /**
-   * @copydoc TensorV2::copyData(const TensorV2 &from)
+   * @copydoc Tensor::copyData(const Tensor &from)
    */
-  void copyData(const TensorV2 &from);
+  void copyData(const Tensor &from);
 
   /**
-   * @copydoc TensorV2::argmax()
+   * @brief      Copy the Tensor
```

Comment:
What does the `stride` stand for? (implicit stride is applied only when data types are heterogeneous) Is it used to copy tensor with different type only? Could you add more exaplanation on the comment?